azure/components/cdn/setup.ftl

```diff
@@ -183,6 +183,43 @@
                     profile=routingRuleResource.Type
                 /]
 
+                [#if deploymentSubsetRequired("epilogue", false)]
+                    [#-- Pages --]
+                    [#if solution.Pages?has_content]
+
+                        [#local setPages = []]
+
+                        [#list solution.Pages as page,path]
+                            [#switch page]
+                                [#case "Root" ]
+                                    [#local setPages += [ "--index-document ${path}" ] ]
+                                [#case "NotFound" ]
+                                    [#local setPages += [ "--404-document ${path}" ] ]
+                                    [#break]
+                                [#default]
+                                    [#break]
+                            [/#switch]
+                        [/#list]
```

Comment:
How would this work if I had multiple SPA's under the same site? won't this override if this is applied to the storage account overall?

Reply:
Also I would have thought this would be a configuration action as a rule on the Frontdoor rather than on the origin